src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmBatchProcessorImpl.java

```diff
@@ -67,23 +66,29 @@ public void processUnstructuredToArm(int armBatchSize) {
         log.info("Started running ARM Batch Push processing at: {}", OffsetDateTime.now());
         ExternalLocationTypeEntity eodSourceLocation = getEodSourceLocation();
 
-        List<ExternalObjectDirectoryEntity> eodsForTransfer = unstructuredToArmHelper
-            .getEodEntitiesToSendToArm(eodSourceLocation,
-                                       EodHelper.armLocation(),
-                                       unstructuredToArmProcessorConfiguration.getMaxResultSize());
-
-
-        log.info("Found {} pending entities to process from source '{}'", eodsForTransfer.size(), eodSourceLocation.getDescription());
-        if (!eodsForTransfer.isEmpty()) {
-            //ARM has a max batch size, so lets loop through the big list creating lots of individual batches for ARM to process separately
-            List<List<ExternalObjectDirectoryEntity>> batchesForArm = ListUtils.partition(eodsForTransfer, armBatchSize);
-            int batchCounter = 1;
-            UserAccountEntity userAccount = userIdentity.getUserAccount();
-            for (List<ExternalObjectDirectoryEntity> eodsForBatch : batchesForArm) {
-                log.info("Creating batch {} out of {}", batchCounter++, batchesForArm.size());
-                createAndSendBatchFile(eodsForBatch, userAccount);
+        int querySize = unstructuredToArmProcessorConfiguration.getMaxArmManifestItems();
+        if (armBatchSize < unstructuredToArmProcessorConfiguration.getMaxArmManifestItems()) {
```

Comment:
`armBatchSize` should be renamed, as its no longer the arm batch size, its the total number of items we intent to process.